crates/vault-registry/src/lib.rs

```diff
@@ -771,6 +781,11 @@ impl<T: Config> Pallet<T> {
                 some_new_threshold.gt(&global_threshold),
                 Error::<T>::ThresholdNotAboveGlobalThreshold
             );
+            Self::deposit_event(Event::<T>::CustomThreshold {
+                vault_id: vault_id,
+                currency_pair: currency_pair,
```

Comment:
```suggestion
                currency_pair: vault_id.currencies,
```

Reply:
Actually as pointed out by @bvotteler this can be removed because it is already included in the `vault_id`